tools/points2labelimage/points2label.py

```diff
@@ -10,6 +11,33 @@
 import skimage.segmentation
 
 
+def geojson_to_tabular(geojson):
+    rows = []
+    for feature in geojson["features"]:
+        name = feature["properties"].get("name")
+        coords = feature["geometry"]["coordinates"][0]
+
+        xs = [pt[0] for pt in coords]
+        ys = [pt[1] for pt in coords]
+
+        x = min(xs)
+        y = min(ys)
+        width = max(xs) - x
+        height = max(ys) - y
```

Comment:
Width or height of 1 means that the rectangle is 1 pixel in width or height:
https://github.com/BMCV/galaxy-image-analysis/blob/1c4cb41b4cd52371bae9d5b9f42110d2d812f7ee/tools/points2labelimage/points2label.py#L90-L93

So if `max(xs) - x` is zero, we want the width to be 1 and not zero:
```suggestion
        width = max(xs) + 1 - x
        height = max(ys) + 1 - y
```

Or am I missing something?